docs/azure/ai/get-started-azure-openai.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Get started with the .NET and Azure Open AI
+description: Get started on how to use the Azure OpenAI with a gpt-35-turbo model, from a simple .NET console application. Get a hiking recommendation from the AI model. It consists of a simple console application, running locally, that will send request to an Azure OpenAI service deployed in your Azure subscription.
+ms.date: 02/20/2024
+ms.topic: get-started
```

Comment:
check to make sure this is a valid ms.topic

Reply:
get-started is present in the list https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#mstopic